source/includes/read/retrieve.rb

```diff
@@ -0,0 +1,47 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+begin
+  client = Mongo::Client.new(uri)
+
+  # start-db-coll
+  database = client.use('sample_training')
+  collection = database[:companies]
+  # end-db-coll
+
+  # Finds one document with a "name" value of "LinkedIn"
+  # start-find-one
+  document = collection.find({ 'name' => 'LinkedIn' }).first
+  puts document
+  # end-find-one
+
+  # Finds documents with a "founded_year" value of 1970
+  # start-find-many
+  results = collection.find({ 'founded_year' => 1970 })
```

Comment:
As before, more idiomatically written as:

```suggestion
  results = collection.find(founded_year: 1970)
```